game/kernel/kmachine.cpp

```diff
@@ -816,6 +817,19 @@ void update_discord_rpc(u32 discord_info) {
   }
 }
 
+u64 filepath_exists(u32 filepath) {
+  auto filepath_str = std::string(Ptr<String>(filepath).c()->data());
+  if (file_util::file_exists(filepath_str)) {
```

Comment:
maybe we can get rid of this function and just do `std::filesystem::exists(filepath_str)`?